packages/apps/human-app/frontend/src/modules/worker/hooks/use-register-address/use-register-address-validation.ts

```diff
@@ -0,0 +1,14 @@
+import { useWalletConnect } from '@/shared/hooks/use-wallet-connect';
+
+export function useRegisterAddressValidation() {
+  const { address } = useWalletConnect();
+
+  const errors = {
```

Comment:
Every time `useRegisterAddressValidation` is called - it will return new `errors` object reference even if input and validation result do not change, hence anything that relies on `errors` reference (e.g. if `errors` is listed as dependencies in `useEffect` hook) won't be able to properly check if it's changed.

Probably `useMemo` should make the deal